packages/oas-utils/src/entities/spec/collection.ts

```diff
@@ -41,6 +41,34 @@ const oasCollectionSchema = z.object({
   // security
 })
 
+/**
+ * All possible integrations
+ *
+ * This is duplicated in @scalar/types/legacy/reference-config.ts so keep that up to date until we centralize
+ */
+const integrations = z.enum([
+  'adonisjs',
+  'docusaurus',
+  'dotnet',
+  'elysiajs',
+  'express',
+  'fastapi',
+  'fastify',
+  'go',
+  'hono',
+  'html',
+  'laravel',
+  'litestar',
+  'nestjs',
+  'nextjs',
+  'nitro',
+  'nuxt',
+  'platformatic',
+  'react',
+  'rust',
+  'vue',
+])
```

Comment:
`null` is valid, too

super annoying to keep those two lists in sync! should we make this just `string | null`? the import modal should gracefully handle other values and fall back to the openapi logo

Reply:
yea thats fine